src/Assertion/AssertionPluginManager.php

```diff
@@ -33,8 +33,11 @@
 {
     protected $instanceOf = AssertionInterface::class;
 
-    public function get($name): AssertionInterface
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
personally, I don't see the point of such API docs. Especially in modern IDE's

Reply:
I added this because of IDE :D without this it complains about uncaught exceptions